pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -248,6 +248,25 @@ func (m *kubeGenericRuntimeManager) generateContainerResources(pod *v1.Pod, cont
 	}
 }
 
+// generateContainerResourcesForUpdate generates platform specific (linux) container resources config for runtime
+// with setting negative values to parameters that will be updated to max.
+func (m *kubeGenericRuntimeManager) generateContainerResourcesForUpdate(pod *v1.Pod, container *v1.Container) *runtimeapi.ContainerResources {
+	containerResources := m.generateContainerResources(pod, container)
+	if containerResources == nil {
+		return nil
+	}
+	if m.cpuCFSQuota && containerResources.Linux.CpuQuota == 0 {
+		containerResources.Linux.CpuQuota = -1
```

Comment:
Is this value `-1` for the same reason that the `maxValue` const you added to `pkg/kubelet/kuberuntime/kuberuntime_manager.go` is `-1`? 

If so, consider putting the `maxValue` const somewhere that it can be shared in both places

Reply:
These are slightly different. This `-1` is passed to the runtime for updating a container resource while the other is passed to `libcontainer/cgroups` for updating pod cgroups. I added a few comments.